src/styles/index.css

```diff
@@ -0,0 +1,312 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+ul,
+li,
+h1,
+p,
+div,
+a {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+.page {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  width: 726px;
+  height: auto;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  place-content: center flex-start;
+}
+
+.calendar__day {
+  width: 100px;
+  height: 100px;
+  border: 1px solid #000;
+  background: #eee;
+  display: none;
```

Comment:
The '.calendar__day' class has 'display: none;' which means that the days will not be visible by default. Ensure this is intentional, as it might hide the calendar days unless overridden by other styles.